### DISCUSSION TITLE
Error with the installtion via GUI Dashboard

### DISCUSSION THREAD

**Discussion by sagarkumar3105** [nocodb/nocodb#668] [Discord Threads]
**Please enter the following details**

![image](https://user-images.githubusercontent.com/74974420/137672742-29426f72-1275-4181-b5cf-2d86ef22d22a.png)

The link given in GUI Dashboard under installation takes you to the local address  not on a web address.

**Comment by o1lab** [comment]
Sorry, its not clear what the issue is and how to reproduce it.
